### PR TITLE
test: add test cases for service/reply

### PR DESCRIPTION
**Pull request by sinchang**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message follows commit guidelines

##### Affected core subsystem(s)
test

##### Description of change
test: add test cases for service/reply

### CODE REVIEW

**Comment by sinchang** [comment]
Ref: #35

File: package.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
     "debug": "egg-bin debug",
     "test": "npm run lint -- --fix && npm run test-local",
     "test-local": "egg-bin test",
+    "test-local:watch": "egg-bin test --watch",
```
Comment (atian25) [inline package.json]: 不用加这些吧，本地要跑就自己:
`npm run test-local -- --watch` 或 `npm egg-bin test --watch`

File: test/app/service/reply.test.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const { app, assert } = require('egg-mock/bootstrap');
+
+describe('test/app/service/reply.test.js', () => {
+  let loginname,
```
Comment (atian25) [inline test/app/service/reply.test.js]: 不要合并，这段按理说过不了 eslint 的

File: test/app/service/reply.test.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    ctx,
+    replyService,
+    replyId,
+    userId,
+    topicId,
+    email;
+  before(async function() {
```
Comment (atian25) [inline test/app/service/reply.test.js]: `async () =>`

**Comment by sinchang** [comment]
与 PR #44  雷同了，不要合。

**Pull request closed without merging**